package.json

```diff
@@ -64,4 +64,4 @@
     "nodeunit": "^0.9.1",
     "verb": "~0.2.0"
   }
-}
+}
```

Comment:
Yeah this has never been [clear to me](https://github.com/lytics/ember-cli-release/issues/21#issuecomment-133497274). Minor and fine either way.